projects/addon-doc/src/components/language-switcher/language-switcher.component.html

```diff
@@ -0,0 +1,37 @@
+<tui-select
+    [tuiTextfieldSize]="size"
+    [tuiDropdownLimitWidth]="limitWidth"
+    [valueContent]="content"
+    [formControl]="language"
+>
+    Language
+    <input
+        tuiTextfield
+        placeholder="Language"
```

Comment:
I don't think we need this since there's never no value selected

Reply:
fixed